security/tempesta/tempesta_lsm.c

```diff
@@ -81,7 +81,7 @@ tempesta_new_clntsk(struct sock *newsk, struct sk_buff *skb)
 }
```

Comment:
Could you please update the year for the file copyright

Reply:
fixed